# HG changeset patch
# User jwe
# Date 747891794 0
#      Mon Sep 13 03:43:14 1993 +0000
# Node ID 4c130d05db388623723762c70890523201bb82ac
# Parent  c9a2dd0150afa5761fec642e8e3dcb5c86473f3a
[project @ 1993-09-13 03:43:14 by jwe]
(panic): Declare volatile.

diff --git a/src/error.cc b/src/error.cc
--- a/src/error.cc
+++ b/src/error.cc
@@ -73,17 +73,17 @@ void
 error (const char *fmt, ...)
 {
   va_list args;
   va_start (args, fmt);
   verror ("error", fmt, args);
   va_end (args);
 }
 
-void
+void volatile
 panic (const char *fmt, ...)
 {
   va_list args;
   va_start (args, fmt);
   verror ("panic", fmt, args);
   va_end (args);
   abort ();
 }
